server/controllers/AuthorController.js

```diff
@@ -0,0 +1,13 @@
+
+/* eslint-disable require-jsdoc */
+const Author = require('../models/Author');
+
+class AuthorController {
+  static async create(req, res) {
```

Comment:
After deliberating on this, let's use basic functions instead of putting them in a class. There's no real value in doing it this way, therefore we go with simplicity.